doc/feature/rescaleOutput.md

```diff
@@ -0,0 +1,80 @@
+DataSource), we need a way to correctly rescale the resulting values, based
+on the input data range and the output type range.
+
+## Requirements
+
+* The internal computations should always be done in floating point values.
+* The data source for Livre must always convert the resulting volume from
+  float to uint8_t values.
+* The voxelize tool will always write a volume in the specified output data
+  type, converting it from float if needed.
+* The process of rescaling the volume data should always consider the output
+  min and max values (known for each data type, e.g. 0-255 for uint8_t), and
+  the input window (min and max values present in the input data).
+* The conversion should always be done at the end of the process, after all the
+  events' contribution has been considered to compute each voxel's value.
+* The min and max values of the input data range should be defined from the
+  beginning, based on biological values and/or actual data. Since this is very
+  hard to know in advance, specially for time-dependent data, a way to manually
+  define this values should be provided. In any case, there should be default
+  values.
+
+## Implementation
+
+Floating point values are already used along the whole process. And the
+voxelize tool is already taking care of the rescaling when the output data type
+is different than float.
+
+In the EventFunctor (base class for the functors) there is a _scale method,
+used on each voxel to scale and clamp the value to the output data range. There
+are several problems with the current implementation (see fieldFunctor, for
+example):
+
+* _scale assumes that the value is always normalized between 0 and 1, which
+  might not be true, depending on the value of the 'magnitude' parameter.
+* In the case of voltages, it's always shifting the event values by
+  brion::MINIMUM_VOLTAGE, to make all the values positive. This is wrong, as
+  soon as we start adding voltages together for the computation of one specific
+  voxel, we need to know their sign.
+
+The new implementation should get rid of these shift and scale operations in the
+functors, and use always the raw values of the report (or the result of the
+corresponding operation for an specific use case, e.g. LFP). All the conversion
+operations should be done only at the end of the process, once we make sure that
+the conversion won't affect the result of the computation.
+
+The 'magnitude' parameter in the input URI is not enough for a correct rescaling
+operation. It was added with the intention to hold the value for the input data
+range, but this assumes that the starting value is 0, incorrect as soon as we
+start dealing with summations of voltages with different signs, for example.
+
+So two new parameters should be added, 'minValue' and 'maxValue'. And
+potentially remove the 'magnitude'. Example:
+
+\verbatim
+voxelize --volume fivoxlfp:///path/to/BlueConfig?report=currentReport,target=targetName,inputMin=-50,inputMax=100
+\endverbatim
```

Comment:
rm verbatim, indent four spaces to get "code" block. same below for outputValue.